recipes/pgure-svt/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pgure-svt" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tjof2/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 33da930fb15f1b7e7ede00f4f04eed0159612fe089d7ef7acca09457427d7a8a
+  patches:
+    - 0001-Remove-setup_requires.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - armadillo
+    - cython
+    - lapack
+    - nlopt
+    - numpy
+    - openblas
+    - pip
+    - python
+  run:
+    - armadillo
+    - lapack
+    - nlopt
+    - {{ pin_compatible('numpy') }}
+    - openblas
+    - python
+
+test:
+  requires:
+    - hyperspy-base
+  imports:
+    - pguresvt.pguresvt
+    - pguresvt.hspysvt
+
+about:
+  home: https://github.com/tjof2/pgure-svt
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+  summary: 'Singular value thresholding for denoising time-resolved microscopy'
+  description: |
+    PGURE-SVT (Poisson-Gaussian Unbiased Risk Estimator - Singular Value Thresholding) is an algorithm designed to denoise image sequences acquired in microscopy. It exploits the correlations between consecutive frames to form low-rank matrices, which are then recovered using a technique known as nuclear norm minimization. An unbiased risk estimator for mixed Poisson-Gaussian noise is used to automate the selection of the regularization parameter, while robust noise and motion estimation maintain broad applicability to many different types of microscopy.
+  doc_url: https://github.com/tjof2/pgure-svt
+  dev_url: https://github.com/tjof2/pgure-svt
+
+extra:
+  recipe-maintainers:
+    - ericpre
+    - tjof2
```

Comment:
@tjof2, can you confirm you are willing to be a maintainer?

Reply:
Yep!